test/resources/qaoa_entry_point.py

```diff
@@ -158,7 +150,3 @@ def cost_function(params):
 
     record_test_metrics('Total', start_time, pl_interface)
     print("Braket Container Run Success")
-
-
-if __name__ == "__main__":
```

Comment:
Why remove this?

Reply:
We're no longer running this module as a script, only importing `entry_point` from here. We only needed this before cause the script itself got run as "main".